sway/ipc-json.c

```diff
@@ -1129,6 +1129,7 @@ json_object *ipc_json_describe_input(struct sway_input_device *device) {
 		json_object *layouts_arr = json_object_new_array();
 		json_object_object_add(object, "xkb_layout_names", layouts_arr);
 
+		if (keymap == NULL) return object;
```

Comment:
nit: coding style, we always use bracketed if blocks, not bracket-less or single-line ifs.

Note that we also cannot just return, as we still need to get past the libinput block to add libinput details to the object. If we consider a NULL keymap a valid state, then we should just make the keymap access conditional.